poetry.lock

```diff
@@ -6342,7 +6335,7 @@ files = [
 
 [[package]]
 name = "swebench"
-version = "1.1.5"
+version = "2.0.2"
```

Comment:
Aha. I wonder if this brought some of the many other updates.

Reply:
Yes i think so! I specifically used `--no-update` here!